<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Puzzle15</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 173, 102, 255), stop:0.55 rgba(200, 125, 51, 255), stop:0.98 rgba(156, 98, 40, 255), stop:1 rgba(0, 0, 0, 0));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btn1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width:4px;
border-radius:10px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width:4px;
border-radius:10px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btn3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width:4px;
border-radius:10px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btn4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width:4px;
border-radius:10px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width:4px;
border-radius:10px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width:4px;
border-radius:10px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btn7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width:4px;
border-radius:10px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btn8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width:4px;
border-radius:10px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width:4px;
border-radius:10px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn10">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width:4px;
border-radius:10px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btn11">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width:4px;
border-radius:10px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btn12">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width:4px;
border-radius:10px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn13">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width:4px;
border-radius:10px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btn14">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width:4px;
border-radius:10px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="btn15">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width:4px;
border-radius:10px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="btn16">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width:4px;
border-radius:10px;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 84, 51, 255), stop:0.55 rgba(107, 64, 27, 255), stop:0.98 rgba(50, 31, 13, 255), stop:1 rgba(0, 0, 0, 0));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>516</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
